src/test/java/dk/sdu/se_f22/productmodule/management/TestSuite.java

```diff
@@ -1,11 +1,12 @@
 package dk.sdu.se_f22.productmodule.management;
+
 import org.junit.platform.suite.api.SelectClasses;
 import org.junit.platform.suite.api.SelectPackages;
 import org.junit.platform.suite.api.Suite;
 
 @Suite
-//@SelectClasses({ProductManagerTest.class, ProductAttributeTest.class, JSONReaderTest.class, ProductTest.class})
-@SelectPackages({"dk.sdu.se_f22.productmodule.management"})
+@SelectClasses({ProductManagerTest.class, ProductAttributeTest.class, JSONReaderTest.class, ProductTest.class})
+//@SelectPackages({"dk.sdu.se_f22.productmodule.management"})
```

Comment:
If commented out shouldn't it be removed?

Reply:
we have been using both for testing